server/hedley/modules/custom/hedley_news/plugins/content_types/news_content/news_content.inc

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Plugin definition.
+ */
+
+$plugin = [
+  'title' => t('Municipality news content'),
+  'description' => t('Municipality news content.'),
+  'category' => t('Hedley'),
+  'render callback' => 'hedley_news_content_type_render',
+  'hook theme' => 'hedley_news_content_content_type_theme',
+  'required context' => new ctools_context_required(t('News'), 'node'),
+  'defaults' => [],
+];
+
+/**
+ * Render callback.
+ */
+function hedley_news_content_type_render($subtype, $conf, $args, $context) {
+  if (empty($context->data)) {
+    return NULL;
+  }
+
```

Comment:
Added in here:
https://github.com/Gizra/Municipality/pull/367/commits/1437d4adb557f3fc501e8913500ef618fcfc650c